src/controllers/agent.js

```diff
@@ -0,0 +1,53 @@
+import Model from '../models/model';
+
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+import dotenv from 'dotenv'
+const agentModel = new Model('agents');
+dotenv.config();
+
+export const signupAgent = async (req, res) => {
+  const { firstName, lastName, password, email, phoneNumber } = req.body;
+  const columns = 'first_name, last_name, password, email, phone_number';
+  const strong = await bcrypt.hash(password, 12);
+  const values = `'${firstName}', '${lastName}', '${strong}', '${email}', '${phoneNumber}'`;
+  try {
+  const checkEmail = await agentModel.select(
+    '*',
+    ` where email = '${email}' `
+  );
+    if (checkEmail.rowCount > 0 ) {
+      return res.status(409).json({ message: 'Account already Exist' });
```

Comment:
Good use of status code `409` is correct, well done.

Reply:
changes worked on.